routes/web.php

```diff
@@ -2,8 +2,21 @@
 
 use Illuminate\Support\Facades\Route;
 use Lightit\Shared\App\Exceptions\InvalidActionException;
+use src\Backoffice\Employee\App\Controllers\ListEmployeesController;
+use src\Backoffice\Employee\App\Controllers\StoreEmployeesController;
+use src\Backoffice\Task\App\Controllers\GetTaskController;
+use src\Backoffice\Task\App\Controllers\ListTasksController;
+use src\Backoffice\Task\App\Controllers\UpsertTaskController;
+
+Route::get('/employees', [ListEmployeesController::class, 'index']);
+Route::post('/employees', [StoreEmployeesController::class, 'store']);
```

Comment:
```suggestion
Route::get('/employees', ListEmployeesController::class);
Route::post('/employees', StoreEmployeeController::class);
```
It's not in the User endpoints so I guess it's not necessary. Also, use singular in the Store endpoints where you are creating just one entity